### DISCUSSION TITLE
Get requests in order to create invoices (without API Keys)

### DISCUSSION THREAD

**Discussion by felipebrunet** [btcpayserver/btcpayserver#6298] [Technical Support]
I used to be able to generate invoices without API Keys and only using GET requests with this URL:
http://{btcpay_server_onion_url}/api/v1/invoices?storeId={store_Id}&price={price_in_USD}&checkoutDesc={sale_motive}&currency=USD
but now it is not working. 
Is this functionality deprecated?

**Comment by pavlenex** [comment]
Hi, what is the use-case here?
